spec/spec_helper.rb

```diff
@@ -1,96 +1,38 @@
 # Run Coverage report
 require 'simplecov'
-SimpleCov.start do
-  add_filter 'spec/dummy'
-  add_group 'Controllers', 'app/controllers'
-  add_group 'Helpers', 'app/helpers'
-  add_group 'Mailers', 'app/mailers'
-  add_group 'Models', 'app/models'
-  add_group 'Views', 'app/views'
-  add_group 'Libraries', 'lib'
-end
+SimpleCov.start 'rails'
 
 # Configure Rails Environment
-ENV['RAILS_ENV'] = 'test'
+ENV['RAILS_ENV'] ||= 'test'
 
-require File.expand_path('../dummy/config/environment.rb',  __FILE__)
+begin
+  require File.expand_path('../dummy/config/environment', __FILE__)
+rescue LoadError
+  puts 'Could not load dummy application. Please ensure you have run `bundle exec rake test_app`'
+  exit
```

Comment:
Please use `abort` instead of `exit`. `exit` by default returns 0 exit code which means success in shell.

http://stackoverflow.com/a/23340693

I'd write this like this:

```ruby
begin
  require File.expand_path('../dummy/config/environment', __FILE__)
rescue LoadError
  abort 'Could not load dummy application. Please ensure you have run `bundle exec rake test_app`'
end
```